core/forage-core-common/src/main/java/org/apache/camel/forage/core/util/config/ConfigStore.java

```diff
@@ -17,6 +17,7 @@
 import java.util.stream.Collectors;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.core.env.Environment;
```

Comment:
@JiriOndrusek @croway should we worry about having this on the forage-common ?

Reply:
An alternative approach I think would be going straight for the application.properties or .env file discoverable from the classpath, which I think covers both runtimes and most runtime specific configuration scenarios for both runtimes.     I don't think it's the most comprehensive because I think both Spring and Quarkus allow you to define custom configuration filenames to load in a variety of ways, but it would allow us to remove any Spring or Quarkus dependencies from core.